scripts/exgfs_wave_prdgen_gridded.sh

```diff
@@ -38,7 +38,7 @@ source "${USHgfs}/wave_domain_grid.sh"
  export DATA=${DATA:-${DATAROOT:?}/${job}.$$}
  mkdir -p "${DATA}"
  cd "${DATA}" || exit 1
```

Comment:
$DATA should have already been created by `jjob_header.sh` and `cd`'d to. Remove these lines and look for similar instances in these scripts.